app/org/sagebionetworks/bridge/dynamodb/DynamoStudy.java

```diff
@@ -299,6 +303,30 @@ public boolean isHealthCodeExportEnabled() {
     public void setHealthCodeExportEnabled(boolean enabled) {
         this.healthCodeExportEnabled = enabled;
     }
+    
+    /** {@inheritDoc} */
+    @DynamoDBMarshalling(marshallerClass = StringIntegerMapMarshaller.class)
+    @Override
+    public Map<String,Integer> getMinSupportedAppVersions() {
+        return minSupportedAppVersions;
+    }
+
+    @Override
+    public void setMinSupportedAppVersions(Map<String,Integer> map) {
+        this.minSupportedAppVersions = (map == null) ? new HashMap<>() : map;
```

Comment:
Yes, if there's no other constructor to call, Java calls the default constructor (always). If there's no default constructor... it adds one that does nothing in the byte code.